### ISSUE TITLE
[SUGGESTION] Make other player's ships targetable with turrets

### ISSUE THREAD

**Issue by matejkar** [tsunamayo/Starship-EVO#3057]
Now that turrets are fixed, i can do capital ship battles (not like i've build 9 capital ships or something). The problem is turrets dont shoot other player's ships.

Now my reccomendation is to set a option to MMB on another player's ship and having options "target" , "lock autopilot" (locking autopilot would work just like on shipyards but would stop close to the player's ship) and "hail" allowing for quick messages such as for example follow me, fall back, hold position, jump to lightcruise, dock (if its a smaller ship). I would also reccomend this for all AI ships, non - aggessive ones that is.

**Comment by tsunamayo** [comment]
Yes, I want to refactor the way we interact with stuff, I keep you posted. Thanks
